samples/sensor/lps22hh_i3c/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_CBPRINTF_FP_SUPPORT=y
+
+CONFIG_SENSOR=y
+CONFIG_LPS22HH=y
+
+CONFIG_LPS22HH_TRIGGER_GLOBAL_THREAD=y
+
+CONFIG_I3C=y
```

Comment:
shouldn't be needed, remove `CONFIG_I3C`

Reply:
removed.